runelite-client/src/main/java/net/runelite/client/plugins/mta/MTAPlugin.java

```diff
@@ -101,6 +101,8 @@ public void shutDown()
 		{
 			eventBus.unregister(room);
 		}
+
+		telekineticRoom.shutDown();
```

Comment:
you probably should never call plugin startUp/shutDown methods directly. Rather, create more descriptive method and call it from both shutDown and from here